lte/gateway/python/scripts/user_trace_cli.py

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import argparse
+import re
+from collections import defaultdict
+from pathlib import Path
+
+
+class UserTrace:
```

Comment:
```suggestion
class UserTrace(object):
```
this will fix remaining pep8 linting error

Reply:
Updated, thanks for the suggestion @ardzoht !